tests/scripts/components-configuration-crypto.sh

```diff
@@ -2028,20 +2062,23 @@ END
         #define PSA_WANT_ALG_SHA3_384  1
         #define PSA_WANT_ALG_SHA3_512  1
         #define PSA_WANT_KEY_TYPE_AES  1
+        #define PSA_WANT_ALG_GCM       1
+        #define PSA_WANT_ALG_HKDF      1
         #define MBEDTLS_PSA_CRYPTO_C
         #define MBEDTLS_CTR_DRBG_C
         #define MBEDTLS_PSA_BUILTIN_GET_ENTROPY
         #define MBEDTLS_SELF_TEST
 END
 
     msg "all loops unrolled"
-    $MAKE_COMMAND clean
-    make -C tests ../tf-psa-crypto/tests/test_suite_shax CFLAGS="-DMBEDTLS_SHA3_THETA_UNROLL=1 -DMBEDTLS_SHA3_PI_UNROLL=1 -DMBEDTLS_SHA3_CHI_UNROLL=1 -DMBEDTLS_SHA3_RHO_UNROLL=1"
+    CFLAGS="-DMBEDTLS_SHA3_THETA_UNROLL=1 -DMBEDTLS_SHA3_PI_UNROLL=1 -DMBEDTLS_SHA3_CHI_UNROLL=1 -DMBEDTLS_SHA3_RHO_UNROLL=1" cmake -D CMAKE_BUILD_TYPE:String=None .
```

Comment:
```suggestion
    CFLAGS="-DMBEDTLS_SHA3_THETA_UNROLL=1 -DMBEDTLS_SHA3_PI_UNROLL=1 -DMBEDTLS_SHA3_CHI_UNROLL=1 -DMBEDTLS_SHA3_RHO_UNROLL=1" cmake  .
```
We have started to remove some "-D CMAKE_BUILD_TYPE:String=None", what about to remove them consistently?